sdk/keyvault/arm-keyvault/package.json

```diff
@@ -25,7 +25,7 @@
     "rollup-plugin-node-resolve": "^3.4.0",
     "uglify-js": "^3.4.9"
   },
-  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/packages/@azure/arm-keyvault",
+  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/sdk/keyvault/arm-keyvault",
```

Comment:
This needs to be reflected in the generator as well. CC @daschult.

Reply:
Created issue for the necessary changes: https://github.com/Azure/autorest.typescript/issues/438

Reply:
https://github.com/Azure/autorest.typescript/pull/439